packages/legend-studio/src/components/editor/edit-panel/uml-editor/ClassEditor.tsx

```diff
@@ -179,15 +184,22 @@ const PropertyBasicEditor = observer(
           </div>
         )}
         {!isIndirectProperty && (
-          <input
-            className="property-basic-editor__name"
-            disabled={isReadOnly}
-            value={property.name}
-            spellCheck={false}
-            onChange={changeValue}
-            placeholder={`Property name`}
-            name={`Property name`}
-          />
+          <div className="input-group__input property-basic-editor__input">
```

Comment:
why do we still need this `wrapper` `<div>`? - please check other palces as well. Thanks!!